app/_shared/components/icons/chevron-right.tsx

```diff
@@ -0,0 +1,18 @@
+import { SVGElementPropsType } from '@/_shared/types/types';
+
+export default function ChevronRight(props: SVGElementPropsType) {
+  return (
+    <svg
+      width="24"
+      height="24"
```

Comment:
Again, this makes icon size is hard-coded to 24x24 size. Use {...props} instead.